scss/tests/mixins/_utilities.test.scss

```diff
@@ -0,0 +1,391 @@
+$prefix: bs-;
+$enable-important-utilities: false;
```

Comment:
It's not the same default as in Bootstrap, but makes the writing of the tests much smoother as there's no `!important` to repeat. It also made more sense to me tin terms of how tests are written to test what happens when the feature gets enabled.